runtime/compiler/env/J2IThunk.cpp

```diff
@@ -107,7 +107,7 @@ char TR_J2IThunkTable::terseTypeChar(char *type)
       {
       case '[':
       case 'L':
-      case 'Q':
+      case 'Q': //not sure if the "if true" part after conditional operator should be changed for 'Q'
```

Comment:
I think it's correct to have the same behaviour for `Q` here as for `L` and `[` - I think it's mapping the type to a 64-bit or 32-bit address.

Reply:
Hmmm - thinking about it again, I guess it's possible that the contents of value types might be passed by value or by reference.  The size of the thing might influence that.  I think it's OK to treat `Q` the same as `L` and `[` for now, but maybe add a TODO to indicate that this might need some further consideration.